### ISSUE TITLE
[BUG] Create a 'Jobs' Page

### ISSUE THREAD

**Issue by yashmandi** [Counselllor/Counsellor-Web#531]
On clicking on 'Jobs', nothing happens, instead it should route me to Jobs Page

Would like to create a 'Job' Page with demo jobs for now

Request you to assign this task to me under GSSoC '24

**Comment by thestarsahil** [comment]
Thanks for letting know to work on Job page !
Please take reference from UI made in readme file 
Good to go !

**Comment by yashmandi** [comment]
> Thanks for letting know to work on Job page ! Please take reference from UI made in readme file Good to go !

hi sahil, may i use tailwindCSS or should i go with normal CSS? because using tailwindCSS would be more effecient

**Comment by thestarsahil** [comment]
Yes go with normal css because tailwind is effecting to all over responsive side

**Comment by Pranali3103** [comment]
I would like to work on this issue @yashmandi

**Comment by thestarsahil** [comment]
Raise the New issue @Pranali3103 for this 
Thanks
